### ISSUE TITLE
problem with workflow

### ISSUE THREAD

**Issue by Parnab03** [vn7n24fzkq/github-profile-summary-cards#193]
**Context**

Your tutorial about how to set up the workflow is too old, I followed the  steps but my readme.md file is not updating.

**Alternatives**

no, I dont know how to fix it!!
![Screenshot (223)](https://github.com/user-attachments/assets/1646ff1b-a4d1-4fba-896a-23eabb4b5b36)

**Comment by Cheezik** [comment]
Hello, im looking into it right now

**Comment by Cheezik** [comment]
Okay so, after some digging up, it seems like an outdated temple was used for the .yml file

Please follow the updated guide here: https://github.com/vn7n24fzkq/github-profile-summary-cards/wiki/Tutorial#create-a-workflow

**Comment by Parnab03** [comment]
if i use those svgs then is it change when new contributions are added in my account?

**Comment by Parnab03** [comment]
also i wnat to set the size of the svg how can i do it then?
I want to make it seems like this how i make it like this with those svgs?
![Screenshot (224)](https://github.com/user-attachments/assets/6d035ba4-d72e-43e8-aa0f-d1e5823f5b81)

**Comment by Cheezik** [comment]
> if i use those svgs then is it change when new contributions are added in my account?

No. if i understand correctly, you are asking if the svgs update with every new contribution. Im personally not 100% if its possible or not, But currently if your using the template from the guide, the Workflow will update every 24h regardless of any changes (new stars on repos, More commits, etc etc)

**Comment by Cheezik** [comment]
> also i wnat to set the size of the svg how can i do it then? I want to make it seems like this how i make it like this with those svgs? ![Screenshot (224)](https://private-user-images.githubusercontent.com/85074838/354808632-6d035ba4-d72e-43e8-aa0f-d1e5823f5b81.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI2ODAwNjAsIm5iZiI6MTcyMjY3OTc2MCwicGF0aCI6Ii84NTA3NDgzOC8zNTQ4MDg2MzItNmQwMzViYTQtZDcyZS00M2U4LWFhMGYtZDFlNTgyM2Y1YjgxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODAzVDEwMDkyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk2NTkzNWJlMDBhMWNhMWZlODQ4NTJlOWU5MjE3ZjY1MjNmNWZjZTA5MDEyNzVkNjMxZjBkNzM4MGEzZTU5MDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hSLN12ZfcSTd9hvZQgQfJ9PisJL8khBPCmHua6sR61k)

I dont think thats currently an option sadly

**Comment by Parnab03** [comment]
acctually done it

**Comment by Cheezik** [comment]
interesting, anyways lmk if you need anything else
